scripts/replicate-build/template/predict.py

```diff
@@ -82,13 +82,13 @@ async def lmtp_inference_stream(self, ops_batch, loop=None):
                         prev_warned.add(local_id)
                     continue
 
-                if response_type != 'TOKEN':
+                if response_type != 'TOKEN' and response_type != 'MSG':
                     print(f'WARNING: Unrecognized response type {response_type!r}; may need to be updated', file=sys.stderr)
 
                 response_data['stream_id'] = orig_id
 
                 yield f'{response_type!s} {json.dumps(response_data)}\n'
-                if response_data.get("finish_reason") is not None:
+                if response_data.get("finish_reason") is not None or response_type == 'MSG':
```

Comment:
@charles-dyfis-net I added a MODEL_INFO command to LMTP, allowing the client to pull basic information on the model that is running. I tested this via `python predicate.py` and given that the changes are minimal, I assume it should be fine. Maybe you can briefly confirm this?

Reply:
Looks right to eyeball it. To test in practice I'd need to rebuild and deploy new images to replicate; as I'm offline (at a cabin with my wife celebrating our 10th anniversary) starting tomorrow, though, it may be the better part of a week before that can happen.

Reply:
Thanks. I just tested it with a deployed model and everything seems to work nicely, except for the model being reported as `checkpoint`. But we can fix this later, protocol-wise the changes will be compatible with the replicate backend.